panels/source/Panels.js

```diff
@@ -736,7 +736,7 @@
 		* @private
 		*/
 		fireTransitionFinish: function () {
-			var t = this.finishTransitionInfo;
+			var t = this.finishTransitionInfo || {fromIndex: null, toIndex: null};
```

Comment:
Probably better to add a couple local variables for fromIndex and toIndex rather than creating another object. Something like:

```
var t = this.finishTransitionInfo,
    fromIndex = t ? t.fromIndex : null,
    toIndex = t ? t.toIndex : null;
```

and then provide those to the object literal on L742.